File: mario.py
from cs50 import get_int

height = 0
while height <= 0 or height > 8:
    height = get_int("Height: ")

for i in range(height):
    for j in range(height):
        if i + j < height - 1:
            print(" ", end="")
        else:
            print("#", end="")
    print()

File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n <= 0 || n > 8);

    for (int i = 1; i <= n; i++)
    {
        for (int j = 1; j <= n; j++)
        {
            if (i + j >= n + 1)
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }
        printf("  ");

        for (int k = 1; k <= n; k++)
        {
            if (k <= i)
            {
                printf("#");
            }
        }
        printf("\n");
    }
}
